src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -195,7 +196,9 @@ private void init() {
     titleTextField.textProperty().addListener((observable, oldValue, newValue) -> {
       preferencesService.getPreferences().getLastGamePrefs().setLastGameTitle(newValue);
       preferencesService.storeInBackground();
+      adjustCreateGameButtonBackgroundColor(newValue);
     });
+    adjustCreateGameButtonBackgroundColor(titleTextField.getText());
```

Comment:
Add a unit test set the text to "" and to "test" and see if the Pseudo class is set correctly.

Reply:
Should be easy to test

Reply:
Resolving this as this was done